README.markdown

```diff
@@ -575,62 +520,34 @@ var faxNumber: Optional<Int>
 ```
 
 * Use convenient Swift Core Geometry methods such as `someFrame.width` or `someframe.midX` to access view's properties.
-* Use `nil` only in method calls or assignments. Do not compare to `nil` in conditional statements.
+* Use `nil` only in method calls or assignments. Avoid comparing to `nil` in conditional statements, unless it increases the visibility.
```

Comment:
Compare to 'nil' in conditional statements, if it increases the readability of your code.